experimental/plugins_network.md

```diff
@@ -0,0 +1,43 @@
+# Experimental: Docker network driver plugins
+
+Docker supports network driver plugins via 
+[LibNetwork](https://github.com/docker/libnetwork). Network driver plugins are 
+implemented as "remote drivers" for LibNetwork, which shares plugin 
+infrastructure with Docker. In effect this means that network driver plugins 
+are activated in the same way as other plugins, and use the same kind of 
+protocol.
+
+## Using network driver plugins
+
+The means of installing and running a network driver plugin will depend on the
+particular plugin.
+
+Once running however, network driver plugins are used just like the built-in
+network drivers: by being mentioned as a driver in network-oriented Docker
+commands. For example,
+
+    docker network create -d weave mynet
+
+(assuming the [Weave](https://github.com/weaveworks/docker-plugin) plugin is
+currently running). Other plugins are listed in [plugins.md](plugins.md)
+
+The network thus created is owned by the plugin, so subsequent commands
+referring to that network will also be run through the plugin.
+
+Plugins might provide their own tools e.g. for checking the status of the
+plugin or for working with network policy. For example 
```

Comment:
am not too sure if this is the right place to mention this side-band management.
Infact, am with the opinion that the very nature of plugins should be plug-and-play & hence any such side-band mechanism will hurt portability. Personally, i will discourage such measures and work together in the community to provide a common abstraction at the plugin layer that will benefit the user and the plugin developers.
In this case, both the `status` and `network policy` are common and must be worked on at the plugin layer.